habana/1.17.1/ubi9-python-3.11/Dockerfile

```
@@ -0,0 +1,47 @@
+
+# Copyright (c) 2024 Habana Labs, Ltd.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+USER root
+
+## Label the image with details required by ODH
+LABEL name="odh-notebook-habana-jupyter-1.17.1-ubi9-python-3.11" \
```

I wonder whether the naming isn't a bit confusing here; let's have the version right after habana word?
```suggestion
LABEL name="odh-notebook-habana-1.17.1-jupyter-ubi9-python-3.11" \
```

similarly for other cases?